org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/ZigBeeThingTypeMatcher.java

```diff
@@ -172,4 +172,30 @@ public RequiredProperty(String name, String value) {
             this.value = value;
         }
     }
+    
+	/**
+	 * Transforms \x&lt;nnnn&gt; occurrences into corresponding utf16 char.
+	 * 
+	 * @param s String that shall be unescaped.
+	 * @return UTF16 string
+	 */
+	private static final String unescape(String s) {
```

Comment:
Why does this need to be static? I can't immediately see a need for that so if it's not required then I'd suggest to remove this.

Reply:
unescape is rather a utility method than a member of TigBeeThingTypeMatcher, as it does not apply to its properties but to a String argument. In this kind of case, I usually create a Util class for holding such methods. Here, with only one util method, keeping it in the user class looked to be the best choice.

Reply:
Ok, but I see it as just another method in the class and I think in general making all (or really, any) methods static is not highly desirable. If this was a helper class where you were providing static methods externally, then I could see some merit, but as it is it's all internal and private.

I'm happy for it to be in the user class - it certainly makes no sense for it to be elsewhere - I just don't see the need to make it static.

Reply:
Fixed. Removed "final" too after having thought about it.